test/scylla-scripts-pytest/pytest.ini

```diff
@@ -0,0 +1,7 @@
+# Pytest configuration file. If we don't have one in this directory,
+# pytest will look for one in our ancestor directories, and may find
+# something irrelevant. So we should have one here, even if empty.
+[pytest]
+# don't use the common test/conftest.py, since those test aren't running against scylla
+args =
+    --confcutdir .
```

Comment:
@kostja is this necessary in order to **not** use ../test.conftest.py? I thought it was the opposite, and code that wants to use it needs "addopts = --confcutdir ..".
If it's necessary, we also need to add it in test/alternator, test/cql-pytest, test/redis, and others.

Reply:
I've added it cause the upper level conftest.py, was bringing in too many imports (like scylla-driver), which I don't need.

as far as I know the default pytest behavior is to lookup the directory structure for plugins (conftest.py), and use them all